types/types.go

```diff
@@ -6,7 +6,7 @@ package types
 import "errors"
 
 var (
-	ErrNotFound = errors.New("log entry not found")
```

Comment:
What do you think about changing this to be something like:

```go
ErrNotFound = raft.ErrLogNotFound
```

That way it's always in sync with whatever the raft library is using.